### PR TITLE
change some stuff

### PR DESCRIPTION
**Pull request by michaelsbradleyjr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ The easiest way to create Ethereum [DApps](https://en.wikipedia.org/wiki/Decentr
 
 If you run into any issues or have feedback, please [file an issue](https://github.com/michaelsbradleyjr/create-embark-dapp/issues/new).
 
+this is stuff that I changed
```
Comment (brainproxy) [inline README.md]: ... yes but is it good?

File: README.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ The easiest way to create Ethereum [DApps](https://en.wikipedia.org/wiki/Decentr
 
 If you run into any issues or have feedback, please [file an issue](https://github.com/michaelsbradleyjr/create-embark-dapp/issues/new).
 
+this is stuff that I changed... yeah, but is it good?
```
Comment (brainproxy) [inline README.md]:
```
yeah but is it really, really good?
```

**Review by brainproxy — COMMENTED** [review]
tell me about it

File: README.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ The easiest way to create Ethereum [DApps](https://en.wikipedia.org/wiki/Decentr
 
 If you run into any issues or have feedback, please [file an issue](https://github.com/michaelsbradleyjr/create-embark-dapp/issues/new).
 
+this is stuff that I changed... yeah, but is it good? ... or does it stink?
```
Comment (brainproxy) [inline README.md]: yeah but... what now?

**Comment by michaelsbradleyjr** [comment]
Was experimenting with PR reviews, comments, etc. using another GitHub account I setup. I didn't realize at the time that I wouldn't be able to wholly delete dummy PRs.

**Pull request closed without merging**